src/com/palmergames/bukkit/towny/TownyMessaging.java

```diff
@@ -47,22 +49,18 @@ public static void sendErrorMsg(String msg) {
 	 * @param msg the message to send
 	 */
 	public static void sendErrorMsg(Object sender, String msg) {
-		boolean isPlayer = false;
 		if (sender instanceof Player) {
-			isPlayer = true;
+			((Player) sender).sendMessage(PREFIX + ChatColor.RED + msg);

```

Comment:
No need for casting here.

Reply:
I can cast the object to a CommandSender at the beginning to avoid casting multiple times, but at least one cast is necessary since the method signature requires an Object